src/runtime/propensity-test.js

```diff
@@ -70,32 +78,10 @@ describes.realWin('Propensity', {}, env => {
     }).throw(/Entitlements must be an Object/);
   });
 
-
-  it('should provide valid event', () => {
```

Comment:
why is this test removed?

Reply:
I reordered the tests to group logical items together.  It was just moved.